configman/tests/test_val_for_configobj.py

```diff
@@ -339,6 +343,100 @@ def test_write_ini_with_reference_value_froms(
             self.assertEqual(expected.strip(), received.strip())
 
         #----------------------------------------------------------------------
+        def test_write_ini_with_reference_value_froms_secrets(
+            self
+        ):
+            n = self._some_namespaces()
+            n.namespace('x1')
+            n.x1.add_option(
+                'password',
+                default='secret "message"',
+                doc='the password',
+                likely_to_be_changed=True,
+                reference_value_from='xxx.yyy'
+            )
+            n.namespace('x2')
+            n.x2.add_option(
+                'password',
+                default='secret "message"',
+                doc='the password',
+                reference_value_from='xxx.yyy'
+            )
```

Comment:
Should we add my suggestion of a `password_hint` option that explicitely sets `secret=False`